pkg/settlement/pseudosettle/pseudosettle.go

```diff
@@ -128,7 +129,7 @@ func (s *Service) Pay(ctx context.Context, peer swarm.Address, amount uint64) er
 	s.logger.Tracef("sending payment message to peer %v of %d", peer, amount)
 	w := protobuf.NewWriter(stream)
 	err = w.WriteMsgWithContext(ctx, &pb.Payment{
-		Amount: amount,
+		Amount: amount.Uint64(),
```

Comment:
consider changing this protocol field for type safety.

Reply:
well, pseudosettle is only used for testing (and only if you run a cluster without swap). in such a setting this should never exceed uint64. anyway we can still do this at a later point.